pkg/controllers/dynakube/istio/reconciler.go

```diff
@@ -2,14 +2,18 @@ package istio
 
 import (
 	"context"
+	"errors"
 	"net"
 
 	dynatracev1beta1 "github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta1/dynakube"
 	dtclient "github.com/Dynatrace/dynatrace-operator/pkg/clients/dynatrace"
 	"github.com/Dynatrace/dynatrace-operator/pkg/controllers/dynakube/connectioninfo/activegate"
 	"github.com/Dynatrace/dynatrace-operator/pkg/controllers/dynakube/connectioninfo/oneagent"
+	"github.com/Dynatrace/dynatrace-operator/pkg/util/conditions"
 	"github.com/Dynatrace/dynatrace-operator/pkg/util/kubeobjects/labels"
-	"github.com/pkg/errors"
+	"github.com/Dynatrace/dynatrace-operator/pkg/util/timeprovider"
+	errorshelper "github.com/pkg/errors"
```

Comment:
pretty sure you can just remove this and use `"errors"` instead

Reply:
the `WithMessage` func is missing in `"errors"` 😭

Reply:
that's why I just left it in... we decided to postpone the change to a different story, but after doing some fast research I found, that there are a bunch of libs, each with advantages and disadvantages. The one we use is pretty simple so I would just keep it for now...